tensorrt_llm/usage/llmapi_config.py

```
@@ -0,0 +1,386 @@
+            "capture_version": CAPTURE_VERSION,
+            "capturable_field_count": state.capturable_field_count,
+            "captured_field_count": len(state.values),
+            "excluded_marked_field_count": state.excluded_marked_field_count,
+            "field_policy_version": FIELD_POLICY_VERSION,
+            "included_field_count": len(state.values),
+            "schema_digest": _schema_digest(llm_args.__class__),
+            "source": CAPTURE_SOURCE,
+            "unsafe_excluded": state.unsafe_excluded,
+        }
+        return _canonical_json(state.values), _canonical_json(metadata)
+    except Exception:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)